src/ua.c

```diff
@@ -1180,6 +1189,7 @@ int ua_alloc(struct ua **uap, const char *aor)
 	struct uri *luri;
 	char *buf = NULL;
 	char *host = NULL;
+	static uint32_t ua_suffix = 0;
```

Comment:
This should work just fine, but I think it is better to move this to the UAG state.
Personally I would prefer that we try to avoid global/static variables.